### ISSUE TITLE
Selectively hiding stereotypes

### ISSUE THREAD

**Issue by mohanr** [plantuml-stdlib/C4-PlantUML#192]
Is it possible to hide the stereotype of `ContainerDb` ? Can it be selective ? Should I create a style for that ?

```
System_Boundary(c1, "Mobile Banking") {
    Person(bankemployee, Operations, "")
    ContainerDb(db, "Database", "Relational Database Schema", "Stores user registration information, hashed authentication credentials, access logs, etc.")
}
```
The other option is this. Is there a 'cylinder' shape ?

```
AddElementTag("storage", $shape=CylinderShape())
Component(fileStorage, "File storage", "techn", $tags="Database")
```

**Comment by kirchsth** [comment]
Hi @mohanr,

**at a) hide ContainerDb stereotype**
with `HIDE_STEREOTYPE()` you can hide all stereotypes.

C4-Plantuml uses skinparam and no styles and I think you cannot combine it. But if you have a working solution please add it.

I tried following selective solutions but non worked
  
```plantuml
@startuml
!include <c4/c4_container.puml>

' hide all stereotypes
' HIDE_STEREOTYPE()

' this call is not database specific, it hides stereotype of the ContainerDb and other Containers too 
' hide <<container>> stereotype

' if you add this `hideStereo` stereotype as tag then then multiple stereotype are defined and hide is not working anymore
hide <<hideStereo>> stereotype

System_Boundary(c1, "Mobile Banking") {
    Person(bankemployee, Operations, "")
    Container(con, "Container", "techn", "Description")
    ContainerDb(db1, "Database with hideStereo", "Relational Database Schema", "Stores ...", $tags="hideStereo")
    ContainerDb(db2, "Database", "Relational Database Schema", "Stores ...")
}
@enduml
```

I think if multiple stereotypes are combined then you cannot hide only one stereotype (details see [Cannot hide specific stereotype, if multiple stereotypes are used](https://forum.plantuml.net/12721/cannot-hide-specific-stereotype-multiple-stereotypes-used?show=12721#q12721)) and C4-PlantUML uses/supports combined stereotypes. 

**at b) cylinder shape**

Based on the discussion in #145, the current mindset is to not implement a new shape. You could use sprites,...

To summarize:

    We should not add (too many) shapes, otherwise we would have to make a very generic implementation and we lose the C4 relation and create a new "Meta" notation.

    As Simon Brown stated in #97

        there is no "C4 model notation" [..] feel free to use whatever shapes/icons you like

    Adding the shape is only half of the solution. To my mind, we should not be adding shapes but Container, Components, etc.


BR Helmut
